app/code/core/Mage/Core/Helper/Abstract.php

```diff
@@ -212,7 +212,7 @@ public function escapeHtml($data, $allowedTags = null)
             }
         } else {
             // process single item
-            if (strlen($data)) {
+            if (is_string($data) && strlen($data)) {
```

Comment:
If `$data` is not a string, the method will end up returning `null`.

Reply:
I did not want to change the return type, just avoid passing null to strlen.